doc/tutorial/text_analytics/solutions/exercise_02_sentiment.py

```diff
@@ -55,7 +55,11 @@
 
     # TASK: print the cross-validated scores for the each parameters set
     # explored by the grid search
-    print(grid_search.grid_scores_)
+    n_candidates = len(grid_search.candidate_params_)
+    test_split_keys = filter(lambda key: key.startswith('test_split_'),
+                             grid_search.results_.keys())
+    for i in range(n_candidates):
+        print(i, list(grid_search.results_[key][i] for key in test_split_keys))
```

Comment:
> I felt this kind of acts like a demonstration on how to extract the cv scores for i-th candidate, which I think is not obvious to a newbie.

Perhaps not, and as I've expressed before the split scores are what I'm least happy with in this representation of results. However, the user rarely requires the split scores, especially now that we calculate standard deviation for them, so it's not really a good way to summarise a grid search. The split scores are there for aggregation.